### ISSUE TITLE
Mobs dont spawn but their sounds still do?

### ISSUE THREAD

**Issue by Drayruk** [nvb-uy/YSNS#14]
MC 1.21.1
Fabric 0.16.9
Mod v1.0.6

The mod works, the mobs I added don't spawn, but their sounds still do. Like I'll hear chickens and cows even when they've been disabled. 
They don't seem to be interacting with the world, example I can hear a sheep eating but the surrounding grass blocks don't change; so the entities are 100% not there.

Any ideas on how to resolve this?

It's like I installed a schizophrenia mod lol

**Comment by ghost** [comment]
yeah, i used YSNS to get rid of bees but the bee hive made weird sounds.

**Comment by Drayruk** [comment]
> yeah, i used YSNS to get rid of bees but the bee hive made weird sounds.

Oh yeah i've ran into that one too on MC 1.20 and 1.21.1. Its weird my guess its how bees are made???

**Comment by SpaceSandwich24** [comment]
> yeah, i used YSNS to get rid of bees but the bee hive made weird sounds.

That's because behives try to spawn a bee every frame and that makes a sound, if you get rid of bees you have to get rid of beehives too.

But this issue isn't exclusive to beehives, the mobs themselves also make sounds and footsteps as if they were only invisible which I think is an issue with the mod

**Comment by nvb-uy** [comment]
Minecraft is weird and won't really sync, for beehives I'd suggest to use block swap and replace them with fresh beehives using the blockstat replacer, that way they will be empty and won't make sounds, or else it'll loop indefinitely as bees won't be able to spawn but the beehive will try to constantly
